tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -0,0 +1,66 @@
+<tool id="targetfinder" name="targetfinder" version="0.1.0">
```

Comment:
the <description> tag is missing. It would be nice to make this a little bit more informative.
E.g. name="RNA target prediction" <description>tool for plants (targetfinder)</description>